target_chains/cosmwasm/tools/src/test.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  ChainIdsTestnet,
+  ChainIdsMainnet,
+  createExecutorForChain,
+} from "./chains-manager/chains";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { Pipeline } from "./pipeline";
+import { getChainConfig } from "./configs";
+import { PythWrapperExecutor, PythWrapperQuerier } from "./pyth-wrapper";
+import { DeploymentType, getPythContractAddress } from "./helper";
+import { PriceServiceConnection } from "@pythnetwork/price-service-client";
+import { CosmwasmQuerier } from "./chains-manager/chain-querier";
+const argv = yargs(hideBin(process.argv))
+  .usage("USAGE: npm run wormhole-stub -- <command>")
+  .option("mnemonic", {
+    type: "string",
+    demandOption: "Please provide the mnemonic",
+  })
+  .option("deploy", {
+    type: "string",
+    desc: "Execute this script for the given networks.",
+    choices: ["mainnet", "testnet-stable", "testnet-edge"],
+    demandOption: "Please provide the deployment type",
+  })
+  .help()
+  .alias("help", "h")
+  .wrap(yargs.terminalWidth())
+  .parseSync();
+
+async function run() {
+  const STORAGE_DIR = `./${argv.deploy}/test-contracts`;
```

Comment:
you've hardcoded a bunch of directories like this in multiple places in this PR. You should extract all of these paths to a single file. that could be a class like FilesystemStore with methods like .getTestContractDir, or it could just be a file with some functions or string constants.